lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -855,7 +855,7 @@ export abstract class ClientApplication {
             ...this.initializeBaseRequest(validatedRequest)
         };
 
-        this.browserStorage.updateCacheEntries(validatedRequest.state, validatedRequest.nonce, validatedRequest.authority);
+        this.browserStorage.updateCacheEntries(validatedRequest.state, validatedRequest.nonce, validatedRequest.authority || this.defaultAuthority.canonicalAuthority);
```

Comment:
`validatedRequest` should already set a default authority, why is this needed?

Reply:
good catch. I've removed it.